src/v2/Apps/Artist/Components/__tests__/ArtistMeta.jest.tsx

```diff
@@ -0,0 +1,354 @@
+import { ArtistMeta_artist } from "v2/__generated__/ArtistMeta_artist.graphql"
```

Comment:
All of this meta stuff is copy pasta from old app. Its mind bogglingly complicated